applicant/views.py

```diff
@@ -6,7 +6,7 @@
 from django.utils import timezone
 from django.contrib.auth import authenticate, login, logout
 from django.contrib.auth.models import User
-from company.models import JobListing, Application, Company
+from company.models import *
```

Comment:
why taka út að lista upp og láta frekar * ??

Reply:
Because i needed much more and couldnt be bothered to type all